packages/remix-react/routeModules.ts

```diff
@@ -23,7 +27,7 @@ export interface RouteModule {
     | V1_HtmlMetaDescriptor
     | V2_MetaFunction
     | V2_HtmlMetaDescriptor[];
-  unstable_shouldReload?: ShouldReloadFunction;
+  shouldRevalidate?: ShouldRevalidateFunction;
```

Comment:
We are not keeping `unstable_shouldReload` - users will have to change to `shouldRevalidate`